utils/bin/lvim.bat

```diff
@@ -0,0 +1,13 @@
+@echo off
```

Comment:
how does this compare to invoking the powershell script directly (see #1893)? 
- we won't have to maintain two different files
- arg parsing might be easier without routing to powershell?

```batch
@echo off
setlocal enabledelayedexpansion
set args=%*
:: replace problem characters in arguments
set args=%args:`"='%
set args=%args:(=``(%
set args=%args:)=``)%
set invalid=`"='
if !args! == !invalid! ( set args= )
powershell -noprofile -ex unrestricted `"& <lvim_ps1_path> $arg %args%;exit `$lastexitcode`"
```